library/swift/UpstreamHttpProtocol.swift

```diff
@@ -3,12 +3,15 @@ import Foundation
 /// Available upstream HTTP protocols.
 @objc
 public enum UpstreamHttpProtocol: Int, CaseIterable {
+  case alpn
```

Comment:
docs for these might be useful - ALPN is probably not something people are familiar with if they don't work in networking much. Also, should we make this the default for requests or will that be a separate PR?

Reply:
Hm. Let's do both the defaulting and docs it in this PR too.

Reply:
Im thinking of adding a section about the upstream protocols here: https://envoy-mobile.github.io/docs/envoy-mobile/latest/api/http.html